docs/blog/posts/2025-02-24-terms-of-service.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Introducing our new Terms of Service
+description: PyPI is formalizing our policies to help us move forward with new services.
+authors:
+  - ewdurbin
+date: 2025-02-25
+tags:
+  - policies
+  - transparency
+---
+
+We're introducing a new
+[Terms of Service](https://policies.python.org/pypi.org/Terms-of-Service/)
+to formalize our relationship to users
+and enable us to move forward with providing new features and services,
+specifically
+[Organization Accounts](https://docs.pypi.org/organization-accounts/).
+
+<!-- more -->
+
+PyPI has had some form of [Terms of Use](https://policies.python.org/pypi.org/Terms-of-Use/)
+document for users since it
+[began accepting uploads in 2005](https://github.com/pypi/legacy/commit/b139c00cfc5794159afb1fc185d77dbc5fc1a2a4#diff-a67499b048e6bb6ef08d44c7a3c541199615b68e3bd153eb0ccedc492e3dec9dR7-R13)
+and has only been updated twice[^1] since.
+These terms have primarily served to protect PyPI
+and the Python Software Foundation (PSF) who operates it.
+
+Over time we have introduced additional policies to protect our users and community
+such as our
+[Code of Conduct](https://policies.python.org/python.org/code-of-conduct/)
+[Privacy Notice](https://policies.python.org/pypi.org/Privacy-Notice/)
+and
+[Acceptable Use Policy](https://policies.python.org/pypi.org/Acceptable-Use-Policy/).
+
+Our new
+[Terms of Service](https://policies.python.org/pypi.org/Terms-of-Service/)
+formalizes our relationship to PyPI users,
+makes protections for the PSF and PyPI users more explicit,
+and establishes terms we need to provide
+[Organization Accounts](https://docs.pypi.org/organization-accounts/)
+to paid
+[Corporate Organizations](https://docs.pypi.org/organization-accounts/pricing-and-payments/#corporate-organizations).
+
+We have worked with our legal team to retain compatibility with the existing
+[Terms of Use](https://policies.python.org/pypi.org/Terms-of-Use/)
+while adding as permissive a set of new terms as possible to ensure that PyPI users 
+and the PSF are protected.
+
+You will notice a banner on login reminding you of these updated terms,
+as well as an email notification to your primary email address if it has been verified.
+These terms will take effect **TBD** and
+your continued use of PyPI after that date constitutes agreement to these new terms.
+
+[^1]: 
+  See these commits for substantive changes since the Terms of Use was introduced:
+  [2009-11-29](https://github.com/pypi/legacy/commit/ddbd32a78a431ab46cad912046c2492998edc618#diff-a6e30135c956f467cffa36eb37a756a53921754d55ddd6ea80d2a0b4c3f4abfaR16-R33)
+  and
+  [2016-12-16](https://github.com/pypi/legacy/commit/f645942c65a372fdacd4d48ffb4afed4502632e8#diff-bbf95bcc6416475537256acea89690f7c6b1f965c0306e9b883813bd3e4f6c10R15-R98).
```

Comment:
Should we provide a diff for the new changes as well? I feel like folks might appreciate that.

Reply:
there's nothing to diff since they're two completely different docs.